db/migrate/20190724113717_remove_cuisine_from_restaurants.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveCuisineFromRestaurants < ActiveRecord::Migration[5.1]
```

Comment:
I would consider to roll back and have a clean migration process (as a best practice)

Reply:
done